content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx

```diff
@@ -133,3 +133,110 @@ prisma.$extends({
 ```
 
 You cannot use `Prisma.getExtensionContext(this)` inside an [`$allModels`](#add-a-custom-method-to-all-models-in-your-schema) structure.
+
+## Advanced type safety on models
+
+<Admonition type = "info">
```

Comment:
I'm adding this comment so that the branch doesn't get accidentally merged before we're ready